formulas/states/saline-grafana/files/mgr-saline-state-jobs.json.jinja

```diff
@@ -406,7 +406,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "max by (fun,test,mods) (salt_state_jobs{instance=\"$server:$saline_port\",status=\"targeted\"})",
+          "expr": "max by (fun,test,mods) (salt_state_jobs{instance=\"$server/saline\",status=\"targeted\"})",
           "format": "table",
```

Comment:
And such changes also doesn't work in my case, but it works fine if left `$saline_port` with the queries and just change it in constant to `80`

Reply:
since it's using https it'll be 443, I suppose.